src/bio/bio_config.c

```diff
@@ -256,10 +255,9 @@ read_file(const char *filename, size_t *size)
 	FILE *file = fopen(filename, "r");
 	void *data;
 
-	if (file == NULL) {
+	if (file == NULL)
```

Comment:
checkpatch will not complain about this case where you put a comment there.  I already don't like our style choice to remove braces as I think it makes code harder to read and edit, but I really would prefer braces there.

Reply:
Do you really want me to re-push to make a change that goes against our coding standards? if we feel strongly then we should change the coding standard.